.github/dependabot.yml

```diff
@@ -0,0 +1,64 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'weekly'
+      # Use Eastern Standard Time (UTC -05:00)
+      timezone: 'America/Toronto'
+    open-pull-requests-limit: 1
+    commit-message:
+      prefix: 'chore: '
+    reviewers:
+      - 'Seneca-CDOT/winter-2021-telescope'
+      - 'c3ho'
+      - 'manekenpix'
+      - 'raygervais'
+      - 'cindyledev'
+    labels:
+      - 'dependencies'
+    # Disable automatic rebasing
+    rebase-strategy: 'disabled'
+  - package-ecosystem: 'npm'
+    directory: '/src/frontend/next'
+    schedule:
+      interval: 'weekly'
+      time: '07:00'
```

Comment:
You don't do time above, should we be the same for all of these?

Reply:
I am looking at https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#scheduletime and can't figure out how `weekly` schedule works with the `time` attribute. They say weekly checks run on Monday at 5 am UTC by default. I guess if we just specify the `America/Toronto` timezone, it would do checks on Mondays at 5 am our local time. Does that work for us @humphd?

Reply:
I don't think we care when it happens, so let's remove time.